test/legacy/runtests.sh

```diff
@@ -32,7 +32,7 @@ fi
 FS_THREAD_TEST="tests/fs_thread_test"
 
 if ! test -x ${FS_THREAD_TEST} ; then
-    FS_THREAD_TEST="tests/fs_thread_test.exe"
+    FS_THREAD_TEST="tests/fs_thread_test"
```

Comment:
This change defeats the purpose of the case. When `tests/fs_thread_test` isn't executable, `tests/fs_thread_test.exe` is supposed to be tried, because when that happens we're probably running tests on Windows or via Wine.

Reply:
Oh, okay.